giddy/ergodic.py

```diff
@@ -1,12 +1,15 @@
 """
-Summary measures for ergodic Markov chains
+Summary measures for ergodic Markov chains.
 """
-__author__ = "Sergio J. Rey <sjsrey@gmail.com>"
+__author__ = "Sergio J. Rey <sjsrey@gmail.com>, Wei Kang <weikang9009@gmail.com>"
 
-__all__ = ['steady_state', 'fmpt', 'var_fmpt']
+__all__ = ['steady_state', 'fmpt', 'var_fmpt',
+           'steady_state_general', 'fmpt_general']
 
 import numpy as np
 import numpy.linalg as la
+import quantecon as qe
```

Comment:
> Not sure if this matters since I'm not a maintainer, but I've got some feedback given I've been tagged. My main concerns are:
> 
> 1. why are we not filling empty states by default?

The tests for spatial dependence in the discrete Markov chains framework (LR and chi-squared tests) only use nonzero pijm (estimated transition probability conditional on the spatial lag) and the degree of freedom is adjusted by excluding the rows full of zeros). I am not sure whether/how filling empty states by default will influence the properties of the tests, but it definitely worth further digging. But for now, I think it makes sense to maintain the original design of the markov/spatial markov classes/functions.

> 2. we should use clearer language than `fill_diagonal`, since this has a pretty different meaning elsewhere.

I agree! Thank you for the great suggestion. `fill_empty_classes` sounds much better!

Reply:
> Interesting. I dig making this a dep!

This is! Serge and I have been thinking about making `quantecon` as a dependency for a while. It has been added to the requirement.txt